docker-compose.yaml

```diff
@@ -40,7 +43,7 @@ services:
       # Allows graphical programs in the container.
       - /tmp/.X11-unix:/tmp/.X11-unix:rw
       - ${XAUTHORITY:-$HOME/.Xauthority}:/root/.Xauthority
-    command: /bin/bash
+    command: sleep infinity
```

Comment:
Did you want this change in as well?

It's very minor, but with `/bin/bash` it makes it so you can do `docker compose up base` and get right into a terminal, instead of having to do `docker compose up base` and then `docker exec -it base bash` in another.

Reply:
Oh interesting... `docker compose up base` drops you into a terminal? For me it just creates an empty session which needs to be force killed on shutdown (hence one more `ctrl-c` smash).

See the 10s kill time:

```
$  docker compose up base demo
[+] Running 2/0
 ✔ Container pyrobosim-demo-1  Created                                                                                            0.0s 
 ✔ Container pyrobosim-base-1  Created                                                                                            0.0s 
Attaching to base-1, demo-1
^CGracefully stopping... (press Ctrl+C again to force)
[+] Stopping 2/2
 ✔ Container pyrobosim-demo-1  Stopped                                                                                            0.1s 
 ✔ Container pyrobosim-base-1  Stopped                                                                                           10.1s 
canceled
```
I'll poke at this...

Reply:
oh wait, no, I think I meant `docker compose run base`